client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -145,7 +133,8 @@ class CommitManager(appId: String, val conf: CelebornConf, lifecycleManager: Lif
                     if (partitionType == PartitionType.MAP) {
```

Comment:
We should avoid check for PartitionType in CommitManager, maybe move such methods into commitHandler, same for incrementInflightNum and decrementInflightNum